infrastructure/.m2/settings.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd">
+    <servers>
+        <server>
+            <id>nexus</id>
+            <username>admin</username>
+            <password>VMware1!</password>
+        </server>
+    </servers>
+    <mirrors>
+        <mirror>
+            <id>nexus</id>
+            <mirrorOf>*</mirrorOf> <!-- Use '*' for all repositories or 'central,snapshots' to specify -->
+            <name>Mirror of Central Repository</name>
+            <url>https://repo1.maven.org/maven2/</url>
+        </mirror>
+    </mirrors>
+    <profiles>
+        <profile>
+            <id>nexus</id>
+            <repositories>
+                <repository>
+                    <id>releases</id>
+                    <name>pscoe</name>
+                    <url>http://infra.corp.local/nexus/repository/maven-releases/</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>false</enabled>
+                    </snapshots>
+                </repository>
```

Comment:
@ivan-zdravkov @ivo-kotev @Michaelpalacce those https alerts should be safe to dismiss in this specific case right? this is adding the basic configuration to get things running and it can later be updated easily based on customer preference

Reply:
@VenelinBakalov, yes exactly. We are overriding this security requirement in the settings.xml itself as well since this is strictly internal traffic. Maybe we should make a note of that in the documentation.

Reply:
I'm 50/50 on the matter. We should be able to serve HTTPS with self-signed certificates. But either work for me. The error itself is whatever in this case, but at the same time https > http even for this